File: web3rpc/rpc-specs/code-samples/python/debug/profiling/startCPUProfile.py
import json
from opensdk.sdk import OpenSDK

host = "https://api.baobab.klaytn.net:8651"

fileName = "cpu.profile"

sdk = OpenSDK(host)
debug_response = sdk.debug.start_cpu_profile(fileName)

print(json.loads(debug_response.response.data))

File: web3rpc/rpc-specs/code-samples/javascript/debug/profilling/startCPUProfile.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");    
    const file = "cpu.profile";

    provider.debug.startCPUProfile(file, {}, (err, data, response) => {
        console.log(data);
    });
}
)()